package.json

```diff
@@ -34,14 +34,13 @@
   "devDependencies": {
     "babel-cli": "^6.4.5",
     "babel-preset-es2015": "^6.3.13",
```

Comment:
Can we drop the entire build step just by removing some es2015 features?  If so that would be awesome.